.travis.yml

```diff
@@ -57,6 +57,8 @@ before_install:
     fi
 
 install:
+  - yarn cache clean
```

Comment:
Why is this change made?

We want to have caching in the CI.

Reply:
this was only to get around travis caching my forked `enhanced-resolve` install (from a previous, not yet working state). will remove this now